Add PfLO docs

Comandeer opened the pull request:
I've added documentation about PfLO feature, it's SDK example and links to it to other "Paste from *" features.

Closes ckeditor/ckeditor4#3797.

jacekbogdanski commented on docs/features/pastefromlibreoffice/README.md:
```diff
@@ -0,0 +1,85 @@
+
+The following sample content from a LibreOffice Writer document:
+
+{@img assets/img/pastefromlibreoffice_01.png 804 A sample LibreOffice Writer document.}
```
This image seems very blurry. It would be nice to have some HIDPI images instead.

Comandeer replied:
These images are hi-dpi… This is how LO looks on my macbook 🤷‍♂ 
<img width="1073" alt="Comparison between Word and LO: Word view is sharp while LO's one – blurry" src="https://user-images.githubusercontent.com/1078728/74725069-a69cd480-523d-11ea-9483-2fa723cfad56.png">

Comandeer replied:
Seems like it will be fixed in 6.4.1: https://bugs.documentfoundation.org/show_bug.cgi?id=122218

jacekbogdanski replied:
That's a bummer. It seems fine on Linux Fedora, but I don't have hidpi, so I can't help much. @AnnaTomanek could you take a look if we will be fine with such quality of the images? They don't look good IMO ;/

jacekbogdanski commented on docs/features/pastefromlibreoffice/README.md:
```diff
@@ -0,0 +1,85 @@
+
+Here is another sample with some more complicated formatting and styling as well as an image:
+
+{@img assets/img/pastefromlibreoffice_03.png 748 A sample LibreOffice Writer document with complex formatting and image.}
```
The same issue, the image is very unreadable.

jacekbogdanski commented on docs/features/pastefromlibreoffice/README.md:
```diff
@@ -0,0 +1,85 @@
+meta-title-short: Paste from LibreOffice
+---
+<!--
+Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
```
Incorrect license header date.

jacekbogdanski commented on docs/sdk/examples/pastefromlibreoffice.html:
```diff
@@ -0,0 +1,295 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
```
Incorrect license header date.

Comandeer commented:
I've changed license headers and added info into "How to" guide. However I didn't change screenshots due to https://github.com/ckeditor/ckeditor4-docs/pull/288#discussion_r380568737

jacekbogdanski commented on docs/sdk/examples/pastefromlibreoffice.html:
```diff
@@ -0,0 +1,295 @@
+				extraAllowedContent: 'img[width,height,align];span{background}',
+
+				// Enabling extra plugins, available in the full-all preset: https://ckeditor.com/cke4/presets
+				extraPlugins: 'colorbutton,font,justify,print,tableresize,uploadimage,uploadfile,pastefromword,liststyle,pagebreak',
```
I'm wondering if we need `pastefromword` here, but probably shouldn't hurt, especially that someone could just open incorrect sample when checking out how PFW works. Maybe it makes sense to add `pastefromgdocs` also?

On the other hand, shouldn't these plugins be added automatically by preset?

AnnaTomanek commented:
It looks like the LO 6.4.1 is out and the issue with the images is solved. We can redo the screenshots as soon as the issues with pasting the sample documents that I reported to @Comandeer are fixed.

Note: The sample document with the complex structure that is available in the repository is not the same one that is being used in screenshots :smile:

f1ames commented:
There is one important thing which I see is still not covered - described in https://github.com/ckeditor/ckeditor4/issues/3797#issuecomment-585152973. Can we mention somwehere in the docs this fact too? WDYT @AnnaTomanek?

If we will be mentioning that it's also good to mention that even though PfLO is not supported, enabling PFW is recommended since it covers many LO cases too.

AnnaTomanek commented:
@f1ames This is actually mentioned in the sample in the box there, but we can also add a similar note in the docs, it's a good idea :+1: . I'll cover it.

f1ames commented:
Rebased onto latest `major`.

AnnaTomanek commented:
Good job @Comandeer for fixing the plugin so quick :grinning:
